### PR TITLE
IBX-431: Allowed to lock & unlock drafts in workflow

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | IBX-431 |
| Versions      | 3.3, master |

Describe lock/unlock draft feature. [Remember to replace screenshots when merging down to 3.3]

### CODE REVIEW

File: docs/publishing/editorial_workflow.md [hunk 1]
```diff
@@ -14,25 +14,55 @@ For example, you can pass content through stages of draft, design and proofreadi
     Contact your administrator or developer team to configure workflows.
     See [developer documentation](https://doc.ibexa.co/en/master/guide/workflow/) for information on how to do it.
 
-You can view Content items which are in different stages under review on the **My dashboard** screen, in the Review queue table.
-The table will only show the Content items that you have permissions to edit.
+## Review queue
+
+You can view Content items which are in different stages under review on the 
+**My dashboard** screen, in the Review queue table.
+The table will only show Content items that your Role has permissions to edit.
```
Comment (DominikaK) [inline docs/publishing/editorial_workflow.md:21]:
```suggestion
The table only shows Content items that your Role has permissions to edit.
```

File: docs/publishing/editorial_workflow.md [hunk 2]
```diff
@@ -14,25 +14,55 @@ For example, you can pass content through stages of draft, design and proofreadi
...
+If your installation is [configured to support draft locking](https://doc.ibexa.co/en/master/guide/workflow/#draft-locking), the table will 
+also inform you whether any reviewers are assigned and have claimed their drafts 
+for review.
```
Comment (DominikaK) [inline docs/publishing/editorial_workflow.md:24]:
```suggestion
If your installation is [configured to support draft locking](https://doc.ibexa.co/en/master/guide/workflow/#draft-locking),
the table also informs you whether any reviewers are assigned and have claimed their drafts for review.
```

File: docs/publishing/editorial_workflow.md [hunk 3]
```diff
@@ -14,25 +14,55 @@ For example, you can pass content through stages of draft, design and proofreadi
...
 
-It lists all the transitions that this content has gone through.
+When you click the **Edit draft** icon and open a Content item that has been sent 
+to you for review, you see the Events timeline that lists all the transitions 
+that this content has gone through.
 
 ![Events timeline](img/workflow_events_timeline.png)
 
-You can view all configured workflows in the Admin Panel by selecting **Workflow**.
+If draft locking is supported, you will also see a message that confirms that the draft is now locked to you.
```
Comment (DominikaK) [inline docs/publishing/editorial_workflow.md:39]:
```suggestion
If draft locking is supported, you also see a message that confirms that the draft is now locked to you.
```

File: docs/publishing/editorial_workflow.md [hunk 4]
```diff
@@ -14,25 +14,55 @@ For example, you can pass content through stages of draft, design and proofreadi
...
+The table will only show Content items that your Role has permissions to edit.
+If your installation is [configured to support draft locking](https://doc.ibexa.co/en/master/guide/workflow/#draft-locking), the table will 
+also inform you whether any reviewers are assigned and have claimed their drafts 
+for review.
 
 ![Review queue in the dashboard](img/dashboard_review_queue.png)
 
-When you edit a Content item that has been sent for review, you will see the Events timeline.
+In the Review queue table, you can access the draft for editing, review its history, 
+and unlock the draft or request that a lock is released.
```
Comment (DominikaK) [inline docs/publishing/editorial_workflow.md:29]:
```suggestion
and unlock the draft or ask to release the lock.
```
...maybe. Subjunctive tends to be hard to read for non-fluent readers

**Review by DominikaK — APPROVED** [review]

**Review by Nattfarinn — APPROVED** [review]

**Pull request merged**